python-plugins/square/plugin/tables/merchants.py

```diff
@@ -0,0 +1,47 @@
+from cloudquery.sdk.schema.table import Table
+import pyarrow as pa
+from typing import Any, Generator, List
+
+from cloudquery.sdk.schema import Table
+from cloudquery.sdk.schema import Column
+from cloudquery.sdk.schema import Table
+from cloudquery.sdk.scheduler import TableResolver
+from plugin.client import Client
+from square.api.merchants_api import MerchantsApi
+
+
+class Merchants(Table):
+    def __init__(self) -> None:
+        super().__init__(
+            "merchants",
+            [
+                Column("id", pa.string(), primary_key=True),
+                Column("business_name", pa.string()),
+                Column("country", pa.string()),
+                Column("language_code", pa.string()),
+                Column("currency", pa.string()),
+                Column("status", pa.string()),
+                Column("main_location_id", pa.string()),
+                Column("created_at", pa.timestamp("ms")),
+            ],
+        )
+
+    @property
+    def resolverClass(self):
```

Comment:
There's already a `resolver` defined in Table
```python
    def resolver(self, client: Client, parent=None) -> Generator[Any]:
        raise NotImplementedError()
```
It fits the signature of `resolve` below, so I can do something like:
```python
    def resolver(self, client: Client, parent=None) -> Generator[Any]:
        return MerchantsResolver(self).resolve(client, parent)
```

But then the `self._scheduler.sync(self._client, resolvers)` is not possible because sync expects `List[TableResolver]` which we now can't have.

Maybe remove resolver abstractclass from Table?

Reply:
where do we have resolver on table? if we have we might want to resolve it as from what I recall it causes bunch of conflicts and circular dependencies.

Reply:
@yevgenypats https://github.com/cloudquery/plugin-sdk-python/blob/main/cloudquery/sdk/schema/table.py#L39

Reply:
Oh seems like old stuff. should remove prob

Reply:
ok fixed here, will update SDK later